README.md

```diff
@@ -33,7 +33,7 @@ remotes::install_github("pharmaverse/admiral")
 
 ## Release Schedule
 
-[{admiral}](https://pharmaverse.github.io/admiral/cran-release/)` releases are targeted for the first Monday of the last month of each quarter.  Pull Requests will be frozen the week before a release.
+[{admiral}](https://pharmaverse.github.io/admiral/cran-release/) releases are targeted for the first Monday of the last month of each quarter.  Pull Requests will be frozen the week before a release.
```

Comment:
```suggestion
[{admiral}](https://pharmaverse.github.io/admiral/cran-release/) releases are targeted for the first Monday of the last month of each quarter.  
```

I feel like we never follow that part of guidance and we should just drop it.

Reply:
Agree, actioned.

Reply:
I agree that we do not need it here because it is not of interest for the users. But we should add it to our release strategy in admiraldev. With our new branching strategy we must freeze all PRs during the release activities. Otherwise, the version on CRAN and the version tagged in github could differ.

Reply:
agree could get a little messy - I made an issue https://github.com/pharmaverse/admiraldev/issues/337